app/jobs/create_uphold_channel_card_job.rb

```diff
@@ -0,0 +1,71 @@
+class CreateUpholdChannelCardJob < ApplicationJob
+  queue_as :default
+
+  def perform(uphold_connection_id:, channel_id:)
+    uphold_connection = UpholdConnection.find(uphold_connection_id)
+    channel = Channel.find(channel_id)
+    return unless uphold_connection&.is_member? && channel&.verified?
+
+    unless uphold_connection.can_create_uphold_cards?
+      Rails.logger.info("Could not create uphold card for channel #{uphold_connection.publisher_id}. Uphold Verified: #{uphold_connection.uphold_verified}")
+      SlackMessenger.new(message: "Could not create uphold card for the channel #{uphold_connection.publisher_id}.", channel: SlackMessenger::ALERTS).perform
+      return
+    end
+
+    upfc = UpholdConnectionForChannel.where(
```

Comment:
It should only find one, so I think there should be some kind of uniqueness constraint somewhere, right?

Reply:
Just added a validation
https://github.com/brave-intl/publishers/pull/2094/commits/e95259299e1a0f477d13b779ab7d0072878f5bd0